libs/output/src/index.ts

```diff
@@ -1,21 +1,22 @@
-import chalk from 'chalk';
+import { styleText } from 'node:util';
 import { EOL } from 'os';
 
 type Colors = 'red' | 'cyan' | 'green' | 'yellow' | 'gray';
 
 type TaskStatus = 'success' | 'failure' | 'skipped';
 
-function isCI() {
-  return (
-    (process.env.CI && process.env.CI !== 'false') || // Drone CI plus others
-    process.env.GITHUB_ACTIONS === 'true' // GitHub Actions
-  );
-}
+// TODO: check to see if we still need this
+// function isCI() {
+//   return (
+//     (process.env.CI && process.env.CI !== 'false') || // Drone CI plus others
+//     process.env.GITHUB_ACTIONS === 'true' // GitHub Actions
+//   );
+// }
 
-if (isCI()) {
-  // Disable coloring when running in CI environments.
-  chalk.level = 0;
-}
+// if (isCI()) {
+//   // Disable coloring when running in CI environments.
+//   process.env.NODE_DISABLE_COLORS = 'true';
+// }
```

Comment:
Maybe remove these comments?

Reply:
Thanks for the reminder Hugo. I was waiting for CI to run to clarify styletext is smart enough to disable colours. 👍